### PR TITLE
Fix canvas rendering bugs caused by issues with gpu rendering kernel

### PR DESCRIPTION
**Pull request by ykevu**
## What
Fixes canvas rendering bugs, namely #366 and #367, which were both caused by issues in the gpu rendering kernel. Specifically, a check is now called to ensure that a value is contained within the bounds of expected values before trying to render the label associated with that value (out of bounds indexing was resulting not in errors but the cells reappearing upon deletion). Additionally, the max number of cells displayed has been increased to 5000.

These changes were tested by Changhua and me and appear to be working properly.

## Why
Users can now render up to 5000 cells at once on a canvas and will not experience bugs when deleting the highest valued cells in a given frame.

### CODE REVIEW

File: backend/deepcell_label/blueprints.py [hunk 1]
```diff
@@ -75,7 +75,9 @@ def create_project():
         )
     labels_url = request.form['labels'] if 'labels' in request.form else None
     axes = request.form['axes'] if 'axes' in request.form else None
-    with tempfile.NamedTemporaryFile(delete=DELETE_TEMP) as image_file, tempfile.NamedTemporaryFile(delete=DELETE_TEMP) as label_file:
+    with tempfile.NamedTemporaryFile(
+        delete=DELETE_TEMP
+    ) as image_file, tempfile.NamedTemporaryFile(delete=DELETE_TEMP) as label_file:
```
Comment (msschwartz21) [inline backend/deepcell_label/blueprints.py:80]: I thought the operating system changes were introduced in another PR. Is this commit left over from old branch history or were there some changes that didn't make it into the last PR?

File: frontend/src/Project/Canvas/LabeledCanvas.js [hunk 2]
```diff
@@ -71,6 +73,7 @@ export const LabeledCanvas = ({ setBitmaps }) => {
         output: [width, height],
         graphical: true,
         dynamicArguments: true,
+        loopMaxIterations: 5000,
```
Comment (msschwartz21) [inline frontend/src/Project/Canvas/LabeledCanvas.js:76]: Can you add a comment mentioning that this controls the max number of cells on the canvas?

**Review by msschwartz21 — COMMENTED** [review]
Two quick questions, but otherwise this looks good to go.

File: backend/deepcell_label/blueprints.py (hunk 1 above)
Comment (ykevu) [reply backend/deepcell_label/blueprints.py:80]: This is left over from old history; I think my branch was not up to date, sorry

**Review by msschwartz21 — APPROVED** [review]

**Pull request merged**